docs/installation-mac.md

```diff
@@ -37,21 +47,19 @@ The collector uses `brew services` the following commands control the collector
 
 ```sh
 # Start the collector
-brew services start observiq/observiq-otel-collector/observiq-otel-collector
+launchctl start com.observiq.collector
 
 # Stop the collector
-brew services stop observiq/observiq-otel-collector/observiq-otel-collector
-
-# Restart the collector
-brew services restart observiq/observiq-otel-collector/observiq-otel-collector
+launchctl stop com.observiq.collector
```

Comment:
What's the context for this change to using launchctl start/stop for service control?

I've seen three different ways to start/stop services in Darwin (the previous brew commands, launchctl start/stop, and also launchctl load/unload, which is used by `kardianos/service` for starting and stopping services), so I just want to understand why launchctl start/stop is preferred here.

Reply:
The brew services commands didn't work and I couldn't find any reason why. Might be something to revisit. I'm not super familiar with launchctl so I'll look into load/unload and see how that differs.

Reply:
@BinaryFissionGames From the launchctl help it describes load and unload as:

```
load            Recommended alternatives: bootstrap | enable. Bootstraps a service or directory of services.
unload          Recommended alternatives: bootout | disable. Unloads a service or directory of services.
```

and start/stop:

```
start           Starts the specified service.
stop            Stops the specified service if it is running.
```

from this I think start/stop is correct as we don't want to bootstrap or unbootstrap the service but rather just turn it on/off.

Reply:
Cool, sounds good